### PR TITLE
front: fix text input alignment

### LINKED ISSUE #5068: front: all input fields are text-right aligned, should text-left by default
### What happened?

The input fields are text right aligned.

Image

### What did you expect to happen?

The input fields are text LEFT aligned, unless exception.

### How can we reproduce it (as minimally and precisely as possible)?

N/A

### What operating system, browser and environment are you using?

- **Browser**: Firefox v109
- **OS**: Arch
- **Env**: Dev

### OSRD version (top right corner `Account` button > `Informations`)

87022614

### PR DESCRIPTION
**Pull request by Akctarus**
close #5068

### CODE REVIEW

**Review by SharglutDev — APPROVED** [review]
LGTM. Tested in local

File: front/src/common/BootstrapSNCF/InputGroupSNCF.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ type Props = {
   min?: number;
   max?: number;
   step?: number | string;
-  textRight?: boolean;
+  textAlign?: 'left' | 'right' | 'center';
```
Comment (Math-R) [inline front/src/common/BootstrapSNCF/InputGroupSNCF.tsx]: Is "center" a wise choice for an input ? i think left and right were sufficient.
Plus left is the native behaviour of alignement so you should have kept it with "textRight" props to handle the only case where it's not left

File: front/src/common/BootstrapSNCF/InputSNCF.tsx [hunk 2]
```diff
@@ -37,7 +37,7 @@ export type InputSNCFProps = {
   step?: number | string;
   isFlex?: boolean;
   condensed?: boolean;
-  textRight?: boolean;
+  textAlign?: 'left' | 'right' | 'center';
```
Comment (Math-R) [inline front/src/common/BootstrapSNCF/InputSNCF.tsx]: same as above

**Review by Math-R — CHANGES_REQUESTED** [review]

File: front/src/common/BootstrapSNCF/InputGroupSNCF.tsx (hunk 1 above)
Comment (SharglutDev) [reply front/src/common/BootstrapSNCF/InputGroupSNCF.tsx]: I think that was a request from @nicolaswurtz

Comment (nicolaswurtz) [reply front/src/common/BootstrapSNCF/InputGroupSNCF.tsx]: Nope, textRight should be false by default, Mathieu is right (lol).

Comment (Akctarus) [reply front/src/common/BootstrapSNCF/InputGroupSNCF.tsx]: Done

File: front/src/common/BootstrapSNCF/InputSNCF.tsx (hunk 2 above)
Comment (Akctarus) [reply front/src/common/BootstrapSNCF/InputSNCF.tsx]: done

**Review by clarani — APPROVED** [review]
LGTM and tested :heavy_check_mark:

**Review by Math-R — APPROVED** [review]

**Pull request merged**